include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -409,8 +419,8 @@ class argument_parser
     {
         if (sub_parser == nullptr)
         {
-            throw design_error("You did not enable subcommand parsing on construction "
-                                      "so you cannot access the sub-parser!");
+            throw design_error("You did not enable subcommand parsing on construction so you cannot access the "
+                               "sub-parser!");
```

Comment:
The message is misleading. I thought I have to set a bool or something to true.
Maybe: 
```suggestion
            throw design_error("No subcommand was provided at the construction of the argument parser!");
```